geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/GatewayReceiverManagementDUnitTest.java

```diff
@@ -105,7 +94,24 @@ public void createGWR() throws Exception {
         .hasStatusCode(ClusterManagementResult.StatusCode.ENTITY_EXISTS)
         .containsStatusMessage("Member(s) server-1 already has this element created");
 
-    assertManagementResult(cms.list(new GatewayReceiverConfig())).isSuccessful()
-        .hasListResult().hasSize(2);
+    ClusterManagementResultAssert<GatewayReceiverConfig, GatewayReceiverInfo> listAssert =
+        assertManagementResult(cms.list(new GatewayReceiverConfig())).isSuccessful();
+    List<ConfigurationResult<GatewayReceiverConfig, GatewayReceiverInfo>> listResult =
+        listAssert.getResult();
+
+    assertThat(listResult).hasSize(2);
+
+    // verify that we have two configurations, but only group1 config has a running gwr
+    for (ConfigurationResult<GatewayReceiverConfig, GatewayReceiverInfo> result : listResult) {
+      if (result.getConfig().getGroup().equals("group1")) {
+        assertThat(result.getRuntimeInfo()).hasSize(1);
+        assertThat(result.getRuntimeInfo().get(0).getPort()).isGreaterThanOrEqualTo(5000);
+        assertThat(result.getRuntimeInfo().get(0).getMemberName()).isEqualTo("server-1");
+      } else {
+        assertThat(result.getConfig().getGroup()).isEqualTo("group2");
+        assertThat(result.getRuntimeInfo()).hasSize(0);
+      }
+    }
   }
+
```

Comment:
any tests for `get`?

Reply:
don't have get end point yet. This story is for list only